specification/data-resource-semantic-conventions.md

```diff
@@ -70,6 +78,22 @@ Labels defining a compute unit (e.g. Container, Process, Lambda Function).
 | container.image.name | Name of the image the container was built on. | `gcr.io/opentelemetry/operator` |
 | container.image.tag | Container image tag. | `0.1` |
 
+
+### Function as a Service
+
+**type:** `faas`
+
+**Description:** An instance of a function that runs on a platform where underlying hardware concerns, such as provisioning and scaling, have been abstracted away by the platform.
+
+| Label  | Description  | Example  | Required? |
+|---|---|---|--|
+| faas.name | The name of the function being executed. | `my-function` | Yes |
+| faas.id | The unique name of the function being executed. <br /> For example, in AWS Lambda this field corresponds to the [ARN] value, in GCP to the URI of the resource, and in Azure to the [FunctionDirectory] field. | `arn:aws:lambda:us-west-2:123456789012:function:my-function` | Yes |
+| faas.version | The version string of the function being executed as defined in [Version Attributes]. | `semver:2.0.0` | No |
+| faas.instance | The execution environment ID as a string. | `my-function:instance-0001` | No |
```

Comment:
What is the use case for this field? Typically, faas instance is not of interest (it might be a short living thingy). The description does not tell about the nature of the ID (uniqueness etc.).